test_regress/t/t_event.pl

```diff
@@ -10,12 +10,17 @@
 
 scenarios(simulator => 1);
 
-compile(
-    );
+if (!$Self->have_coroutines) {
+    skip("No coroutine support");
+}
```

Comment:
This test didn't need any timing features before, and you haven't changed the .v file, so while does this needed a timing and no-timing split?

Reply:
Sorry, fixed.